scripts/file-entities.php.in

```diff
@@ -14,7 +14,7 @@
   | Authors:    Hartmut Holzgraefe <hholzgra@php.net>                    |
   |             Gabor Hojtsy <goba@php.net>                              |
   +----------------------------------------------------------------------+
-  
+
```

Comment:
Nit: not sure if there are any style rules regarding commits, but otherwise I would suggest to keep removing of trailing newlines to a separate commit/PR (if doing it at all), so the diff is more focused.

Reply:
I'll remove the whitespace changes from this PR.